src/index.js

```diff
@@ -351,21 +355,24 @@ class Firestore extends commonGrpc.Service {
         this,
         ResourcePath.fromSlashSeparatedString(documentOrName)
       );
-      document.readTime = DocumentSnapshot.toISOTime(readTime);
     } else {
       document.ref = new DocumentReference(
         this,
         ResourcePath.fromSlashSeparatedString(documentOrName.name)
       );
```

Comment:
Maybe add a comment something like:

// Cloud Functions calls us with API JSON format data, so we must be sure to convert it.